webapp/channels/src/components/admin_console/ldap_wizard/ldap_helpers.tsx

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import type {MessageDescriptor} from 'react-intl';
+
+import WithTooltip from 'components/with_tooltip';
+
+import type {GeneralSettingProps} from './ldap_wizard';
+
+import SchemaText from '../schema_text';
+
+/**
+ * LDAP-specific component for rendering help text with hover tooltip
+ */
+export const LDAPHelpTextWithHover: React.FC<{
```

Comment:
@hmhealey it's up to you if you want to fold this into the regular help text. For now I'm keeping it separate for simplicity, but 0/5.

Reply:
If you mean folding this into `renderLDAPSettingHelpText`, I like how it looks now.

If you meant making it so that the non-LDAP version of the `HelpText` component also supported the "more info" , I'm also 0/5 on doing that now since most of the logic is still in `SchemaText`, so let's just leave it as-is